src/configs/algo_config.py

```diff
@@ -31,6 +31,9 @@ def get_malicious_types(malicious_config_list: List[ConfigType]) -> Dict[str, st
 traditional_fl: ConfigType = {
     # Collaboration setup
     "algo": "fedavg",
+    "exp_type": "",
+    # Learning setup
+    "epochs": 50,
```

Comment:
I believe I had removed `epochs` in favor of `rounds`, any reason to bring it back?
Also, `exp_type` is already defined in sys_config, isn't it?

Reply:
`exp_type` is not defined in sys_config, but we can skip here as well